### ISSUE TITLE
ete3 and CodingQuarry not installed

### ISSUE THREAD

**Issue by mswyers** [nextgenusfs/funannotate#249]
**Are you using the latest release?**
``$ brew upgrade funannotate``

``Error: nextgenusfs/tap/funannotate 1.5.1 already installed``

**Describe the bug**
Funannotate not finding ete3 or CodingQuarry after latest linuxbrew update. Also not sure when/where CodingQuarry is installed in the readthedocs.io as I haven't seen it before. Also, funannotate check shows my version as 1.5.0, but my brew shows funannotate as being 1.5.1?

```
$ pip install ete3
Requirement already satisfied: ete3 in /home/linuxbrew/.linuxbrew/lib/python2.7/site-packages (3.1.1)

```

**What command did you issue?**
```
$ funannotate check
-------------------------------------------------------
Checking dependencies for funannotate v1.5.0
-------------------------------------------------------
To print all dependencies and versions: funannotate check --show-versions

You are running Python v 2.7.15. Now checking python packages...
All 11 python packages installed


You are running Perl v 5.028001. Now checking perl modules...
All 27 Perl modules installed


Checking external dependencies...
Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/bin/ete3", line 7, in <module>
    from ete3.tools.ete import main
  File "/home/linuxbrew/.linuxbrew/Cellar/python@2/2.7.15_1/lib/python2.7/site-packages/ete3/tools/ete.py", line 55, in <module>
    from . import (ete_split, ete_expand, ete_annotate, ete_ncbiquery, ete_view,
  File "/home/linuxbrew/.linuxbrew/Cellar/python@2/2.7.15_1/lib/python2.7/site-packages/ete3/tools/ete_view.py", line 48, in <module>
    from .. import (Tree, PhyloTree, TextFace, RectFace, faces, TreeStyle, CircleFace, AttrFace,
ImportError: cannot import name TextFace
	ERROR: CodingQuarry not installed
	ERROR: ete3 not installed
Checking Environmental Variables...
All 7 environmental variables are set
```

**Logfiles**
Please provide relavent log files of the error. 

**OS/Install Information**
 -  output of `funannotate check --show-versions`

```
$funannotate check --show-versions
-------------------------------------------------------
Checking dependencies for funannotate v1.5.0
-------------------------------------------------------
You are running Python v 2.7.15. Now checking python packages...
biopython: 1.73
goatools: 0.8.12
matplotlib: 2.2.3
natsort: 5.5.0
numpy: 1.15.4
pandas: 0.23.4
psutil: 5.4.8
requests: 2.19.1
scikit-learn: 0.20.2
scipy: 1.1.0
seaborn: 0.9.0
All 11 python packages installed


You are running Perl v 5.028001. Now checking perl modules...
Bio::Perl: 1.007002
Carp: 1.50
Clone: 0.41
DBD::SQLite: 1.62
DBD::mysql: 4.049
DBI: 1.642
DB_File: 1.84
Data::Dumper: 2.173
File::Basename: 2.85
File::Which: 1.23
Getopt::Long: 2.5
Hash::Merge: 0.300
JSON: 4.00
LWP::UserAgent: 6.36
Logger::Simple: 2.0
POSIX: 1.84
Parallel::ForkManager: 2.02
Pod::Usage: 1.69
Scalar::Util::Numeric: 0.40
Storable: 3.08
Text::Soundex: 3.05
Thread::Queue: 3.13
Tie::File: 1.02
URI::Escape: 3.31
YAML: 1.27
threads: 2.22
threads::shared: 1.59
All 27 Perl modules installed


Checking external dependencies...
Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/bin/ete3", line 7, in <module>
    from ete3.tools.ete import main
  File "/home/linuxbrew/.linuxbrew/Cellar/python@2/2.7.15_1/lib/python2.7/site-packages/ete3/tools/ete.py", line 55, in <module>
    from . import (ete_split, ete_expand, ete_annotate, ete_ncbiquery, ete_view,
  File "/home/linuxbrew/.linuxbrew/Cellar/python@2/2.7.15_1/lib/python2.7/site-packages/ete3/tools/ete_view.py", line 48, in <module>
    from .. import (Tree, PhyloTree, TextFace, RectFace, faces, TreeStyle, CircleFace, AttrFace,
ImportError: cannot import name TextFace
RepeatMasker: RepeatMasker 4.0.7
RepeatModeler: RepeatModeler 1.0.8
Trinity: 2.8.3
augustus: 3.3.2
bamtools: bamtools 2.5.1
bedtools: bedtools v2.27.1
blat: BLAT v36
diamond: diamond 0.9.24
emapper.py: emapper-1.0.3
exonerate: exonerate 2.2.0
fasta: no way to determine
gmap: 2018-07-04
gmes_petap.pl: 4.38
hisat2: 2.1.0
hmmscan: HMMER 3.2.1 (June 2018)
hmmsearch: HMMER 3.2.1 (June 2018)
java: 9.0.4
kallisto: 0.44.0
mafft: v7.407 (2018/Jul/23)
makeblastdb: makeblastdb 2.8.1+
minimap2: 2.14-r883
nucmer: 3.1
pslCDnaFilter: no way to determine
rmblastn: rmblastn 2.2.27+
samtools: samtools 1.9
stringtie: 1.3.4d
tRNAscan-SE: 2.0 (December 2017)
tbl2asn: unknown, likely 25.3
tblastn: tblastn 2.8.1+
trimal: trimAl v1.4.rev15 build[2013-12-17]
	ERROR: CodingQuarry not installed
	ERROR: ete3 not installed
Checking Environmental Variables...
$FUNANNOTATE_DB=/home/seq/fundb
$PASAHOME=/home/seq/PASApipeline-v2.3.3
$TRINITYHOME=/home/linuxbrew/.linuxbrew/opt/trinity
$EVM_HOME=/home/linuxbrew/.linuxbrew/opt/evidencemodeler
$AUGUSTUS_CONFIG_PATH=/home/linuxbrew/.linuxbrew/opt/augustus/config
$GENEMARK_PATH=/home/seq/gmes_petap
$BAMTOOLS_PATH=/home/linuxbrew/.linuxbrew/opt/bamtools
All 7 environmental variables are set
```

**Comment by nextgenusfs** [comment]
You don't need `CodingQuarry` it is only used if you have RNA-seq data.  You can also get by (most likely) without ete3 as it is used during `funannotate compare`. 

I forgot to update the version to `1.5.1` in the `funannotate` script, thus it isn't displaying correct version.  This will be fixed in next release. 

The issue with ete3 is related to the PyQt version -- can read more here: https://github.com/etetoolkit/ete/issues/195

**Comment by estolle** [comment]
I had the same issue with ete3. I just downgraded ete3 and this seems to satisfy funannotate. However there are some RuntimeWarnings thrown by ete3 now, so I am not sure if it will function properly.

$sudo pip uninstall ete3
$sudo pip install ete3==3.0.0b35  (latest version pip seems to have available before 3.1.1)

$ete3 -h
/usr/local/lib/python2.7/dist-packages/scipy/special/__init__.py:640: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from ._ufuncs import *
/usr/local/lib/python2.7/dist-packages/scipy/linalg/basic.py:17: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from ._solve_toeplitz import levinson
/usr/local/lib/python2.7/dist-packages/scipy/linalg/__init__.py:202: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from ._decomp_update import *
/usr/local/lib/python2.7/dist-packages/scipy/special/_ellip_harm.py:7: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from ._ellip_harm_2 import _ellipsoid, _ellipsoid_norm
/usr/local/lib/python2.7/dist-packages/scipy/sparse/lil.py:16: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from . import _csparsetools
/usr/local/lib/python2.7/dist-packages/scipy/optimize/_trlib/__init__.py:1: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from ._trlib import TRLIBQuadraticSubproblem
/usr/local/lib/python2.7/dist-packages/scipy/optimize/_numdiff.py:8: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from ._group_columns import group_dense, group_sparse
/usr/local/lib/python2.7/dist-packages/scipy/interpolate/_bsplines.py:9: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from . import _bspl
/usr/local/lib/python2.7/dist-packages/scipy/spatial/__init__.py:94: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from .ckdtree import *
/usr/local/lib/python2.7/dist-packages/scipy/spatial/__init__.py:95: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from .qhull import *
/usr/local/lib/python2.7/dist-packages/scipy/spatial/_spherical_voronoi.py:18: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from . import _voronoi
/usr/local/lib/python2.7/dist-packages/scipy/spatial/distance.py:121: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from . import _hausdorff
/usr/local/lib/python2.7/dist-packages/scipy/stats/_continuous_distns.py:17: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  from . import _stats
usage: ete3 [-h]
            {mod,extract,annotate,compare,view,ncbiquery,generate,evol,maptrees,build,version,upgrade-external-tools}
            ...

optional arguments:
  -h, --help            show this help message and exit

AVAILABLE PROGRAMS:
  {mod,extract,annotate,compare,view,ncbiquery,generate,evol,maptrees,build,version,upgrade-external-tools}
